example/src/screens/calendars.js

```diff
@@ -357,6 +357,40 @@ const CalendarsScreen = () => {
     );
   };
 
+  const renderCalendarWithCustomHeader = () => {
+    const CustomHeader = React.forwardRef((props, ref) => {
+      return (
+        <View ref={ref} style={{
+          backgroundColor: '#FCC',
+          flexDirection: 'row',
+          justifyContent: "space-around",
+          marginHorizontal: -4,
+          padding: 8
+        }}>
+          <Text>This is a custom header!</Text>
+          <TouchableOpacity onPress={() => console.log("Tapped!")}><Text>Tap Me</Text></TouchableOpacity>
+        </View>
+      );
+    });
+
+    return (
+      <Fragment>
+        <Text style={styles.text}>Calendar with custom header component</Text>
+        <Calendar
+          testID={`custom-header-calendar`}
```

Comment:
If you don't have tests for this calendar example you can remove 'testID' prop as there is no need for it, otherwise, add it to `testIDs.calendar' constants and add a test in calendars.spec.js

Reply:
Done in 00e09ad

Reply:
Sorry if I wasn't clear, there is a test that scrolls the calendars and checks if the last one is visible. That means that the last calendar example should have `testID={testIDs.calendars.LAST}`. So, you need to move your example to be one before last or move the 'testID' from the previous last calendar example to yours. Please run the tests before you submit to make sure it all checks out. Thanks! And sorry for the inconvenience.